README.md

```diff
@@ -281,3 +281,33 @@ The test doesn't require any special setup. The instructions to run it are insid
 [utam-js]: https://www.github.com/salesforce/utam-js
 [utam-doc]: https://utam.dev
 [utam-java-recipes]: https://www.github.com/salesforce/utam-java-recipes
+
+## Run Salesforce Mobile test
+
+- Follow the instructions at [Get Started for Mobile](https://utam.dev/guide/get_started_utam#get-started-for-mobile) to set up your local simulator/emulator.
+- Make sure [Appium](https://github.com/appium/appium#readme) and [node.js](https://nodejs.org/en/) are installed on your machine.
+- Update the wdio configuration file:
+For an iOS test, update `wdio.conf.ios.js` file to configure, test device name and os version, the full path for the test application:
+
+```js
+'appium:deviceName': 'iPhone 12',
+'appium:app': '<path to iOS test app>',
+'appium:platformVersion': '15.2',
+```
+
+For an Android test, update `wdio.conf.android.js` file to configure the application bundleid, the full path for the test application and application initial activity:
```

Comment:
```suggestion
For an Android test, update the `wdio.conf.android.js` file to configure the application bundleid, the full path for the test application, and the application's initial activity:
```

Reply:
Is application bundleid = appium:deviceName or appium:appPackage? It would be helpful to list the corresponding prop in parenthesis in the intro text like in the previous example.

We should also mention the appium:deviceName in the intro text.

I'm not sure what the "application's initial activity" means. How is it used in the test?